src/helpers/figures/charts/abstract_chart.ts

```diff
@@ -94,4 +99,19 @@ export abstract class AbstractChart {
    * Extract the ChartCreationContext of the chart
    */
   abstract getContextCreation(): ChartCreationContext;
+
+  protected getCommonAttributesForExcel(
```

Comment:
Not 100% sure that it should be a method in AbstractChart, it's not relevant/correct for scorecards/gauge charts. Probably should be a helper ? Or at least mention the fact that it's only for charts with datasets in the method name